wrappers/nodejs/binding.gyp

```diff
@@ -45,15 +45,18 @@
               }
             },
             "libraries": [
-              "<(module_root_dir)/../../build/Release/realsense2.lib",
+              "<(module_root_dir)/../../build/Debug/realsense2.lib",
```

Comment:
We need to think how to resolve it in a more generic way. Relying on having CMake output under `build` is a bit problematic (this is the recommendation, but it is up to the user to decide)
Additionally, we don't really know if the user has Debug or Release built

Reply:
Yes, it is a TODO in our list.